src/App.js

```diff
@@ -1,13 +1,57 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import './App.css';
 import axios from 'axios';
+import Pokemon from "./Pokemon";
 
 function App() {
-  return (
+    const [pokemonData, setPokemonData] = useState([]);
```

Comment:
Uiteindelijk is dit niet van het type array, ik zou dan als default waarde `undefined` of `{}` aanhouden